Marlin/ubl_G29.cpp

```diff
@@ -984,6 +996,10 @@
 
       if (do_ubl_mesh_map) ubl.display_map(map_type);  // show user where we're probing
 
+
+      while (ubl_lcd_clicked()) delay(50);;  // wait for user to release encoder wheel
```

Comment:
It looks like your keyboard is malfunctioning. Doubled-up returns, spaces, and other characters keep getting inserted.